src/main.ts

```diff
@@ -17,7 +20,7 @@ async function bootstrap() {
 
   const config = new DocumentBuilder()
     .setTitle('Safe Client Gateway')
-    .setVersion(process.env.npm_package_version ?? '')
+    .setVersion(configService.get('about.version') ?? '')
```

Comment:
I fixed this all well, as it is closely related to the purpose of the PR. Note this wasn't working since we abandoned `npm` as build tool in favor of `yarn`. More context [here](https://github.com/yarnpkg/berry/issues/1659) 

Another besides reading `package.json` directly could be to use [this library](https://github.com/Ayc0/yarn-plugin-envs), but I think it doesn't adds a lot of value for our purposes.

Reply:
> I fixed this also, as it is closely related to the purpose of the PR. Note this wasn't working since we abandoned `npm` as build tool in favor of `yarn`. More context [here](https://github.com/yarnpkg/berry/issues/1659)

For me it seems to be working with `yarn` 🤔 

> Another option, besides reading `package.json` directly, could be to use [this library](https://github.com/Ayc0/yarn-plugin-envs), but I think it doesn't add a lot of value for our purposes.

I would not go with this approach. It's an external dependency (and we are trying to keep those to a minimum) and does more things that we currently require (this should not be complex).